support/Pipelines/EccentricityControl/EccentricityControl.py

```diff
@@ -120,7 +120,10 @@ def compute_separation(h5_file, subfile_name_aha, subfile_name_ahb):
         )
 
     # Compute separation
-    num_obs = min(len(ObjectA_centers[:, 0]), len(ObjectB_centers[:, 0]))
+    num_obs = len(ObjectA_centers[:, 0])
+
+    if len(ObjectB_centers[:, 0]) < num_obs:
+        num_obs = len(ObjectB_centers[:, 0])
```

Comment:
Please drop all changes to this file.

I think what happened is you copied this script a while ago, some updates were made since then, and now you copied your version back into this one. For example, in this line here you fixed the `num_obs`, and I also fixed it in a slightly different way with the `min()` function.

Reply:
how do i do this?

Reply:
ok!